vformer/encoder/vanilla.py

```diff
@@ -6,11 +6,23 @@
 
 
 class VanillaEncoder(nn.Module):
-    def __init__(self, latent_dim, depth, heads, dim_head, mlp_dim, p_dropout=0.0):
+    """
+    class VanillaEncoder:
+    Inputs-
+    --------------
+    latent_dim: embeding dimension
+    depth: number of feedforward and attention blocks in encoding
+    heads: number of heads in self attention block
+    dim_head: head dimensions
+    mlp_dim: dimension of hidden linear layer for the feedforward network
+    p_dropout: probability for the dropout layer
+
+    Forward method returns:
```

Comment:
This should go in the docstrings for the forward method and/or in the general info about the class